_quarto.yml

```
@@ -9,6 +9,12 @@ website:
   site-url: "https://sprout.seedcase-project.org/"
   repo-url: "https://github.com/seedcase-project/seedcase-sprout"
   page-navigation: true
+  body-header: |
+    ::: {.callout-warning appearance="default" icon="true"}
+    🚧 Sprout is still in very active development and is evolving quickly, so
+    documentation and functionality may or may not work as described or intended
+    and could have substantial changes 🚧
```

```suggestion
    🚧 Sprout is still in active development and evolving quickly, so the
    documentation and functionality may not work as described
    and could undergo substantial changes 🚧
```
Just to simplify/shorten it a bit? :)